src/python/pants/backend/python/tasks/resolve_requirements.py

```diff
@@ -13,12 +13,16 @@
 class ResolveRequirements(ResolveRequirementsTaskBase):
   """Resolve external Python requirements."""
   REQUIREMENTS_PEX = 'python_requirements_pex'
+  PREPEND_REQUIREMENTS_PEX = 'prepended_requirements_pex'
 
   options_scope = 'resolve-requirements'
 
   @classmethod
   def product_types(cls):
-    return [cls.REQUIREMENTS_PEX]
+    return [
+      cls.REQUIREMENTS_PEX,
+      cls.PREPEND_REQUIREMENTS_PEX,
+    ]
```

Comment:
How come we're returning two PEXes now? I was expecting to still use one Pex and that changing `PEX_PATH` would let Pex resolves things correctly. This is just an inclination though and not a fact, so I could be wrong here.